frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Button as MuiButton, makeStyles } from "@material-ui/core";
+
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        margin: theme.spacing(0.5)
+    },
+    label: {
+        textTransform: 'none'
+    }
+}))
+
+export default function Button(props) {
```

Comment:
Why do we need a whole separate component to set default values on a Button? Why not just set the values wanted directly on the button where it is used? If specifying all the options is repetitive, you can always put the options in an object and spread the object to provide the props.